Vagrantfile

```diff
@@ -1,5 +1,18 @@
+# Vagrant > 1.8.1 is required due to
+# https://github.com/mitchellh/vagrant/issues/6793
+Vagrant.require_version ">= 1.8.3"
```

Comment:
Never mind, the required version changed but the comment stayed the same (1.8.1) which threw me off.